lib/cf/applications/src/test/api-test.js

```diff
@@ -83,11 +83,11 @@ describe('Admin API', () => {
           applications: {
             config: {
               secured: false,
-              minIntervalTime: 5000,
```

Comment:
Does it make sense to add `API_PAGE_SIZE` to the list of variables we test for change?